app/views/works/index.html.erb

```diff
@@ -0,0 +1,41 @@
+<% books = @works.where(category: "book") %>
```

Comment:
This is a great place to make use of the `top_ten` method from your model so you don't put logic in the view.